prediction_market_agent/agents/think_thoroughly_agent/think_thoroughly_agent.py

```diff
@@ -432,19 +431,13 @@ def generate_prediction_for_one_outcome(
                 for s, a in previous_scenarios_and_answers
             )
 
-        try:
-            output: ProbabilisticAnswer = crew.kickoff(inputs=inputs)
-        except (APIError, HTTPError) as e:
-            logger.error(
-                f"Could not retrieve response from the model provider because of {e}"
-            )
-            return None
+        output: ProbabilisticAnswer = crew.kickoff(inputs=inputs)
```

Comment:
Instead of try-excepting granually, it try-excepts the whole `generate_prediction_for_one_outcome` later in this PR